test-upsert-setup.sql

```diff
@@ -0,0 +1,33 @@
+DROP TABLE IF EXISTS upsert_test_multi_unique; 
+CREATE TABLE upsert_test_multi_unique(time timestamp NOT NULL, temp float, color text);
+CREATE INDEX multi_time_idx ON upsert_test_multi_unique (time);
+CREATE UNIQUE INDEX multi_time_temp_idx ON upsert_test_multi_unique (time, temp);
+CREATE UNIQUE INDEX multi_time_color_idx ON upsert_test_multi_unique (time, color);
+INSERT INTO upsert_test_multi_unique VALUES ('2017-01-20T09:00:01', 25.9, 'yellow');
+INSERT INTO upsert_test_multi_unique VALUES ('2017-01-21T09:00:01', 25.9, 'yellow');
+INSERT INTO upsert_test_multi_unique VALUES ('2017-01-20T09:00:01', 23.5, 'brown');
+
+DROP TABLE IF EXISTS ht_upsert_test_multi_unique; 
+CREATE TABLE ht_upsert_test_multi_unique(time timestamp NOT NULL, temp float, color text);
+SELECT create_hypertable('ht_upsert_test_multi_unique', 'time');
+CREATE UNIQUE INDEX ht_multi_time_temp_idx ON ht_upsert_test_multi_unique (time, temp);
+CREATE UNIQUE INDEX ht_multi_time_color_idx ON ht_upsert_test_multi_unique (time, color);
+INSERT INTO ht_upsert_test_multi_unique VALUES ('2017-01-20T09:00:01', 25.9, 'yellow');
+INSERT INTO ht_upsert_test_multi_unique VALUES ('2017-01-21T09:00:01', 25.9, 'yellow');
+INSERT INTO ht_upsert_test_multi_unique VALUES ('2017-01-20T09:00:01', 23.5, 'brown');
+
+ --UPDATE ht_upsert_test_multi_unique  SET temp = 23.5 where time ='2017-01-20T09:00:01' and color = 'yellow';
```

Comment:
Why's this commented out?

Reply:
oh. that whole file shouldn't even be in there....